docs/guides/MONITORING_QUICKSTART.md

```
@@ -0,0 +1,828 @@
+curl -s https://grafana.com/api/dashboards/12345/revisions/1/download \
+  -o grafana/provisioning/dashboards/json/my-dashboard.json
+
+# Grafana 재시작
+docker-compose restart grafana
+```
+
+---
+
+## 4. 기본 알림 설정
+
+**(준비 중 - Phase 1 다음 단계)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)